packages/tracing-internal/test/browser/metrics/index.test.ts

```diff
@@ -182,15 +206,21 @@ describe('_addResourceSpans', () => {
       decodedBodySize: 2147483647,
     };
 
-    _addResourceSpans(transaction, entry, '/assets/to/css', 100, 23, 345);
+    _addResourceSpans(transaction, entry, resourceEntryName, 100, 23, 345);
 
     // eslint-disable-next-line @typescript-eslint/unbound-method, deprecation/deprecation
     expect(transaction.startChild).toHaveBeenCalledTimes(1);
     // eslint-disable-next-line @typescript-eslint/unbound-method, deprecation/deprecation
     expect(transaction.startChild).toHaveBeenLastCalledWith(
       expect.objectContaining({
-        data: {},
-      }),
+        data: { "server.address": "example.com", "url.same_origin": true, "url.scheme": "https" },
+        description: "/assets/to/css",
+        endTimestamp: 468,
+        op: "resource.css",
+        origin: "auto.resource.browser.metrics",
+        startTimestamp: 445
+      }
```

Comment:
@AbhiPrasad just wanted to double check, this would be the expect behaviour for this test right? 
The span gets added with all properties except the resource size ones?

Reply:
yup that's what I expect, looks good!